workers/unity/Assets/Playground/Scripts/MonoBehaviours/RotationBehaviour.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Generated.Improbable.Transform;
+using Improbable.Gdk.Core;
+using UnityEngine;
+using Quaternion = Generated.Improbable.Transform.Quaternion;
+using UnityQuaternion = UnityEngine.Quaternion;
+using Transform = Generated.Improbable.Transform.Transform;
+
+public class RotationBehaviour : MonoBehaviour
+{
+    [Require] private Transform.Writer writer;
+
+    private UnityQuaternion uRot;
+
+    void Update()
+    {
+        Quaternion rot = writer.Data.Rotation;
```

Comment:
To make this function look prettier I recommend to use conversion methods, or something similar to the [Partials](https://github.com/spatialos/UnitySDK/tree/master/Assets/Plugins/Improbable/.UnityPartials) we have in the SDK